test/shuffling_test.py

```diff
@@ -48,6 +49,64 @@ def test_seed(self):
         new = np.load("Be_REshuffled1.out.npy")
         assert np.isclose(np.sum(np.abs(old-new)), 0.0, atol=accuracy)
 
+    def test_seed_openpmd(self):
+        """Test that the shuffling is handled correctly internally."""
+        test_parameters = mala.Parameters()
+        test_parameters.openpmd_configuration = {
+            'adios2': {
+                'engine': {
+                    'parameters': {
+                        'StatsLevel': 1
+                    }
+                }
+            }
+        }
```

Comment:
Given that we are using HDF5, we can remove the ADIOS2 config here and everywhere else. I'll push a commit.